system-addon/content-src/lib/snippets.js

```diff
@@ -35,6 +35,7 @@ export class SnippetsMap extends Map {
 
   clear() {
     super.clear();
+    this._dispatch(ac.AlsoToMain({type: at.SNIPPETS_BLOCKLIST_UPDATED, data: []}));
```

Comment:
This isn't necessarily what we want to do here, you might have an out-of-date indexedDB version or something on about:home/about:newtab that shouldn't necesarrily affect the main store